master/buildbot/test/unit/test_stats_service.py

```diff
@@ -154,9 +164,9 @@ def test_influx_service_not_inited(self):
         svc.thd_postStatsValue("test", "test", "test")
         self.assertLogged("Service.*not initialized")
 
-    def test_storage_backend_base(self):
-        # Just initlize and run thd_postStatsValue
-        svc = StatsStorageBase()
+    def test_storage_backend_base_failure_on_init(self):
+        svc = DummyStatsStorageBase()
```

Comment:
So this is a positive test, not a negative one, right?  You defined a class that defines the required abstract method.  What about one more test where the class does not define the required abstract method?